.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+## Description
+- Description of chang
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Typo in change